src/DynamoCoreWpf/Views/Preview/Watch3DSettingsControl.xaml

```diff
@@ -13,6 +13,7 @@
                 <TextBox Width="50" Text="{Binding LightAzimuthDegrees, Mode=TwoWay}"/>
                 <Label Content="Elevation:"/>
                 <TextBox Width="50" Text="{Binding LightElevationDegrees, Mode=TwoWay}"/>
+                <Button Content="Select" Command="{Binding TestSelectionCommand}"></Button>
```

Comment:
This is part of the Watch3DSettingsControl which is only available when enabled in the Debug menu, which is only available when Dynamo is built in Debug mode. So yes, the control appears in the UI, but it only appears in debug mode.